ipyleaflet/leaflet.py

```diff
@@ -384,6 +388,28 @@ class Heatmap(RasterLayer):
     gradient = Dict({0.4: 'blue', 0.6: 'cyan', 0.7: 'lime', 0.8: 'yellow', 1.0: 'red'}).tag(sync=True, o=True)
 
 
+class VectorTileLayer(Layer):
+    _view_name = Unicode('LeafletVectorTileLayerView').tag(sync=True)
+    _model_name = Unicode('LeafletVectorTileLayerModel').tag(sync=True)
+
+    url = Unicode().tag(sync=True)
+    options = Dict().tag(sync=True)
```

Comment:
This will conflict with the superclass's `options` attribute. You will need to remove it

Reply:
But as I need the options to be a Dict were as `Layer` class has options trait which is as `List`

Reply:
The dict is computed for you in `this.get_options()` JavaScript side.

Reply:
Ok I get it now